templates/cephtoolbox.yaml

```diff
@@ -7,3 +8,4 @@ metadata:
     argocd.argoproj.io/sync-wave: "10"
 spec:
   enableCephTools: {{ .Values.toolbox.enableCephTools }}
```

Comment:
Starting OCS 4.15, the enableCephTools property was moved from the [OCSInitializationSpec](https://github.com/red-hat-storage/ocs-operator/blob/rhodf-4.15/api/v1/ocsinitialization_types.go#L33) to the [StorageClusterSpec](https://github.com/red-hat-storage/ocs-operator/blob/rhodf-4.15/api/v1/storagecluster_types.go#L106). This feels like a source for future bugs. :-)

Reply:
Agreed, thanks!